platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummaryTest.kt

```diff
@@ -55,7 +56,7 @@ class ConfigurationCacheProblemsSummaryTest {
             subject.onProblem(buildLogicProblem("build.gradle", "failure"), ProblemSeverity.Deferred)
         )
         assertThat(
-            subject.get().uniqueProblemCount,
```

Comment:
❓ `uniqueProblemCount` became `reportUniqueProblemCount` in this PR. Did we change our test assertions to console problems intentionally?

Reply:
Good catch. I changed the tests to inspect both report and console unique counts